tools/build.py

```diff
@@ -18,6 +19,10 @@
         sys.exit(1)
     ninja_args = ["-C", build_path()] + ninja_args
 
+package_json_env = {
+    "TYPESCRIPT_VERSION": package_json.get_typescript_version()
+}
```

Comment:
I don’t like that this depends on special behavior in build.py.
Ideally people would be able to build deno using gn and ninja - build.py is just to make things more ergonomic.
The way I was imagining this done was to print the versions from JS. The V8 version and Deno version can be passed in the Start message.